conda_smithy/configure_feedstock.py

```diff
@@ -102,10 +105,10 @@ def render_circle(jinja_env, forge_config, forge_dir):
 
 @contextmanager
 def fudge_subdir(subdir):
-    orig = conda_build.metadata.cc.subdir
-    conda_build.metadata.cc.subdir = subdir
+    orig = cc.subdir
+    cc.subdir = subdir
     yield
-    conda_build.metadata.cc.subdir = orig
+    cc.subdir = orig
```

Comment:
Right. So that suggests that we are not able to masquerade as one OS (e.g. Linux) when using another OS (e.g. Mac or Win). This is the problem that I was experiencing as well. Namely the re-rendering was wrong. If you have any ideas why it is re-rendering incorrectly now and how we can fix that, that would be very helpful. Personally I'm at a loss.